bin/yaml/cpp.yaml

```diff
@@ -327,23 +336,34 @@ compilers:
             - 4.3.0
             - 4.4.0
             - 4.6.0
+        s390x:
+          arch_prefix: "{subdir}-ibm-linux-gnu"
+          check_exe: "{arch_prefix}/bin/{arch_prefix}-g++ --version"
+          subdir: s390x
+          targets:
+            - name: 11.2.0
         powerpc:
           arch_prefix: "{subdir}-unknown-linux-gnu"
           check_exe: "{arch_prefix}/bin/{arch_prefix}-g++ --version"
-          targets:
-            - name: 4.8.5
-              subdir: powerpc
-              check_exe: "{arch_prefix}/bin/g++ --version"
-            - name: at12
-              subdir: powerpc64
-            - name: at13
-              subdir: powerpc64
-            - name: 6.3.0
-              subdir: powerpc64le
-            - name: at12
-              subdir: powerpc64le
-            - name: at13
-              subdir: powerpc64le
+          powerpc:
+            subdir: powerpc
+            targets:
+              - name: 4.8.5
+                check_exe: "{arch_prefix}/bin/g++ --version"
+              - name: 11.2.0
+          powerpc64:
+            subdir: powerpc64
+            targets:
+              - name: at12
```

Comment:
FWIW as a shortcut if you only need `- name: BLAH` you can use `- BLAH` . I'm lazy :D

Reply:
Didn't know that, thanks ! But there's a lot of hidden magic in these yaml files... Baby steps :D. I'll apply this if you find it better.

Reply:
No problem! The idea was to make the common cases super simple. But the docs are non-existent...so..no wonder you didn't know that :D sorry!

Reply:
Done ! Thanks for the cheat-code ↑↑↓↓←→←→BA